js/chat/chat.ts

```diff
@@ -262,6 +265,7 @@ class ChatInput extends LightElement {
 }
 
 class ChatContainer extends LightElement {
+  @property({ attribute: "icon-assistant" }) iconAssistant = "";
```

Comment:
I don't feel strongly, but have been using `snake_case` for other attributes

Reply:
Yeah, I was a little confused about this actually because it's `icon_assistant` in the `Tag()` call. Are you explicitly forcing snake case for `content_type` somewhere?

I'd say my preference for kebab case attributes trends toward strong (I didn't care enough to figure out why it's `content_type` and not `content-type`, but if I definitely wanted to change it when I saw it).

Reply:
I'm not sure I understand the question and the proposal here. Btw, since it's probably important context, shiny this behavior when writing tags:

```python
>>> from shiny import ui
>>> ui.div(foo_bar = "baz")
<div foo-bar="baz"></div>
>>> ui.div(fooBar = "baz")
<div fooBar="baz"></div>
```

Reply:
Yeah, this is why I was surprised you have underscores in the attributes. So I used `Tag(..., icon_assistant=str(icon_assistant))` in Python, which creates `icon-assistant="...html..."`.

But you're saying you used snake_case for attributes like `content_type`, which means you had to do something special on the Python side, right?

As far as proposals, I'd prefer we pick one of these choices in order from first to last preference:

1. `kebab-case` for attributes
2. `camelCase` for attributes if we want to signal that they're reflected attributes
3. `snake_case` for attributes if we want to signal that they came from Python

(I could be convinced into 2 or 3 if we have some guidelines for consistency, otherwise I'd much rather all attributes in `kebab-case`.)

Reply:
I'm still a bit confused, can we be a bit more specific? AFAIK, I'm using `Tag(content_type="")` in Python which serializes to `<tag content-type="">` in HTML. In the TS, the property name is `content_type`, but the attribute name is `content-type`. Are you proposing `kebab-case` everywhere (even for the TS property names)?

Reply:
Ahh, I overlooked the message aspect 🤦 .

Would it make sense to you if `createElement()` inside of `js/utils/_utils.ts` replaced `_` with `-` (similar to what htmltools in Python does)?